1_antenatal_care.do

```diff
@@ -47,7 +47,7 @@ order *,sequential
 	*c_anc_eff: Effective ANC (4+ antenatal care visits, any skilled provider, blood pressure, blood and urine samples) of births in last 2 years
 	egen anc_blood = rowtotal(m42c m42d m42e),mi
 	gen c_anc_eff = (c_anc == 1 & anc_skill>0 & anc_blood == 3) 
-	replace c_anc_eff = . if c_anc ==. |  anc_skill==. | anc_blood ==.

```

Comment:
The definition of c_anc_eff changed? So blood pressure is not considered in the effective ANC?

Reply:
No change of definition. Because there are filter questions for blood pressure, we cannot say if anc_blood=. then c_anc_eff =.

Reply:
The missing is for the cases where: All non-missing conditions “YES” and 1 or more missing answers on any of the conditions missing OR time of pregnancy missing. 

So anc_blood is a condition?